### ISSUE TITLE
Request for more examples

### ISSUE THREAD

**Issue by yarnball** [mvader/react-categorized-tag-input#18]
Hi,

Great work on this- looks amazing.

However, I haven't been able to integrate it into my own project.

I have written:

```
import TagInput from 'react-categorized-tag-input';

render: function(){
        return (
        <TagInput categories={myCategories} addNew={true}
    transformTag={tagTransformer}
    onBlur={onBlur}
    onChange={onChange} />
        <Jumbotron>
```

This gave me the error `myCategories is not defined`.

So I tried defining myCategories- but this led to the error `ReferenceError: tagTransformer`.

```
var myCategories = [
{
  title: 'String to used to identify the tag',
  category: 'id of the category for the tag'

}
];
```

Could you please provide some direct for usage in a project? Thanks

**Comment by erizocosmico** [comment]
There is an example here https://github.com/mvader/react-categorized-tag-input/blob/master/index.js, which is the code that runs at http://mvader.me/react-categorized-tag-input/

**Comment by yarnball** [comment]
Awesome- thanks for that. Another two simple questions- 
1. Can I make the drop-down of autocomplete clickable (so I can see a list without typing)? 
2. Can I change the auto-complete length to 1 character?

**Comment by erizocosmico** [comment]
answer to both is no. right now that functionality is not configurable, but you can send a PR with it if you need it

**Comment by yarnball** [comment]
Ok- does sending a PR mean I have to figure out how to configure it? Sorry, new here!

**Comment by erizocosmico** [comment]
It means if you need that feature and feel like I'd make sense in the library you can make a fork, implement that functionality there and then make a Pull Request to this repo. And if it is accepted it will be published then to the npm registry.

Feel free to ask if you have any doubts on the process!

**Comment by yarnball** [comment]
I wish I was more skilled- I will try. I am very new to js.

It was very easy to make it auto complete with less characters- I changed line 97 in src/CatagorizedTagInput.jsx from:

```
    if (input.length === 1) {
```

to 

```
    if (input.length === 0) {
```

Where would I go about tiggering the drop down without the user input? Surely something with onClick on the input field?

I found (in "src/CatagorizedTagInput.jsx")

  openPanel() {
    this.setState({ panelOpened: true });
  },

So surely we can "trigger" that some how?
